Gemfile

```diff
@@ -8,10 +8,14 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 gem 'active_model_serializers', '~> 0.10.2'
 gem 'bootsnap', '>= 1.4.2', require: false
+gem 'rack-cors', require: 'rack/cors'
```

Comment:
Why are you adding `rack-cors` in this chore?

Reply:
Because we though now would be a good time to add it. But after a group discussion we decided to wait until we're actually receiving data from another application.